Failing Web API tests

Hi there!

I really enjoy OpenFisca, but I recently encountered an issue.

### Here is what I did:

```
make test
```

### Here is what I expected to happen:

```
508 passed, 1 xfailed, 32 warnings in 30.34s
```

### Here is what actually happened:

```
========================================================================================= short test summary info ==========================================================================================
FAILED tests/web_api/test_variables.py::test_variable_formula_content - assert 'def formula(...me_tax_rate\n' == 'def formula(...me_tax_rate\n'
FAILED tests/web_api/test_variables.py::test_variable_documentation - AssertionError: assert '\nHousing al... necessary.\n' == '\nTo compute... necessary.\n'
========================================================================== 2 failed, 506 passed, 1 xfailed, 32 warnings in 49.60s ==========================================================================
make: *** [test] Error 1
```

### Here is data (or links to it) that can help you reproduce this issue:


## Context

Some Web API tests rely on specifics of openfisca/country-template implementation, specifically literal strings - the objective is to be sure we can obtain de string literals of variables and their documentation via the Web API.

Or, as we might expect, that's fragile, as any change to the string representation of those elements will make the tests fail.

The present failing tests were introduced by openfisca/country-template#97

I identify more as a:

- Developer _(I create tools that use the existing OpenFisca code)_.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,10 +1,17 @@
 # Changelog
 
+### 35.1.1 [#981](https://github.com/openfisca/openfisca-core/pull/981)
+
+#### Technical changes
+
+- Fix false negative web API test following an update in the country template used for testing.
+
 ### 35.1.0 [#973](https://github.com/openfisca/openfisca-core/pull/973)
 
 #### Technical changes
 
 - Extend assert_near so it is able to compare dates.
+
 ### 35.0.5 [#974](https://github.com/openfisca/openfisca-core/pull/974)
 
 #### Technical changes

--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,7 @@ api_requirements = [
 
 dev_requirements = [
     'autopep8 >= 1.4.0, < 1.6.0',
-    'flake8 >= 3.7.0, < 3.9.0',
+    'flake8 >= 3.9.0, < 4.0.0',
     'flake8-bugbear >= 19.3.0, < 20.0.0',
     'flake8-print >= 3.1.0, < 4.0.0',
     'pytest-cov >= 2.6.1, < 3.0.0',

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ dev_requirements = [
 
 setup(
     name = 'OpenFisca-Core',
-    version = '35.1.0',
+    version = '35.1.1',
     author = 'OpenFisca Team',
     author_email = 'contact@openfisca.org',
     classifiers = [

--- a/tests/web_api/test_variables.py
+++ b/tests/web_api/test_variables.py
@@ -94,7 +94,7 @@ def test_variable_formula_github_link():
 
 
 def test_variable_formula_content():
-    formula_code = "def formula(person, period, parameters):\n    return person('salary', period) * parameters(period).taxes.income_tax_rate\n"
+    formula_code = "def formula(person, period, parameters):\n    \"\"\"\n    Income tax.\n\n    The formula to compute the income tax for a given person at a given period\n    \"\"\"\n    return person(\"salary\", period) * parameters(period).taxes.income_tax_rate\n"
     assert variable['formulas']['0001-01-01']['content'] == formula_code
 
 
@@ -157,4 +157,4 @@ def test_variable_documentation():
     variable = json.loads(response.data.decode('utf-8'))
     assert variable['documentation'] == "This allowance was introduced on the 1st of Jan 1980.\nIt disappeared in Dec 2016."
 
-    assert variable['formulas']['1980-01-01']['documentation'] == "\nTo compute this allowance, the 'rent' value must be provided for the same month, but 'housing_occupancy_status' is not necessary.\n"
+    assert variable['formulas']['1980-01-01']['documentation'] == "\nHousing allowance.\n\nThis allowance was introduced on the 1st of Jan 1980.\nCalculating it before this date will always return the variable default value, 0.\n\nTo compute this allowance, the 'rent' value must be provided for the same month,\nbut 'housing_occupancy_status' is not necessary.\n"

